### ISSUE TITLE
Use GrabDragInteraction for the balloon

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#436]
I would like to use the GrabDragInteraction in BASE. Partly to review https://github.com/phetsims/scenery-phet/issues/447, but also because there is a shiny new common type that generalizes this ineteraction.

Porting directly means that a few things will change in BASE related to the accessible labels, so this issue is to track progress and changes. Aterward we can review what is acceptable to change, and possibly what additional options should be added to GrabDragInteraction.

**Comment by jessegreenberg** [comment]
Something is off with the transform when I use the `dragCueNode` option
![0f4ac63ce4a4d22e911f933836f475ec](https://user-images.githubusercontent.com/6396244/50314482-00a8db00-047d-11e9-83f4-d5840875f3cc.gif)

**Comment by jessegreenberg** [comment]
Was happening because the BalloonInteractionCueNode was updating the position of the letters in their parent coordinate frame while the interaction (which was added to the focus overlay) was being transformed in the global coordinate frame.

**Comment by jessegreenberg** [comment]
So the only two changes that BASE got in the design from the common code component are
 - The accessible name for the draggable balloon changed from "Yellow Balloon" to "Movable Yellow Balloon"
 - The WASD/Arrow key drag cue node will be hidden on key up rather than on key down, so the interaction cue remains visible for some movement.

@terracoda do these changes seem OK?

**Comment by terracoda** [comment]
@jessegreenberg, the changes to the Balloon are nice and quite acceptable! Apologies for my late reply on this issue!

**Comment by terracoda** [comment]
@jessegreenberg, I think it is safe to close this issue, but I'll leave that to you.

**Comment by terracoda** [comment]
Closing issue!
